packages/amplify-graphql-transformer-interfaces/src/transformer-context/transform-parameters.ts

```diff
@@ -7,15 +7,21 @@
 export type TransformParameters = {
   // General Model Params
   shouldDeepMergeDirectiveConfigDefaults: boolean;
+  disableResolverDeduping: boolean;
+  sandboxModeEnabled: boolean;
 
   // Auth Params
   useSubUsernameForDefaultIdentityClaim: boolean;
   populateOwnerFieldForStaticGroupAuth: boolean;
+  suppressApiKeyGeneration: boolean;
 
   // Index Params
   secondaryKeyAsGSI: boolean;
   enableAutoIndexQueryNames: boolean;
 
   // Relational Params
   respectPrimaryKeyAttributesOnConnectionField: boolean;
+
+  // Search Params
+  enableNodeToNodeEncryption: boolean;
```

Comment:
This param here doesn't necessarily say that it corresponds to the searchable config. Maybe something like `enableSearchNodeToNodeEncryption` will make it more clear? If the customers are shown appropriate doc string, that should be okay.